kb/all-the-ways-to-run-a-cluster.md

```diff
@@ -5,15 +5,80 @@ tags:
   - cluster
   - server
   - how-to
-date: 2022-10-20T00:00:00Z
+date: 2023-03-09T00:00:00Z
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 There are many ways to run a [Temporal Cluster](/clusters) on your own.
 However, the right way for you depends entirely on your use case and where you plan to run it.
 This article aims to maintain a comprehensive list of all the ways we know of.
 
 <!-- truncate -->
 
+### Temporal CLI
+
+Temporal CLI is a distribution of Temporal that runs as a single process with zero runtime dependencies.
+It supports persistence to disk and in-memory mode through SQLite.
+
+**Install and start the Temporal CLI**
+
+The following steps start and run a Temporal Cluster.
+
+1. Choose one of the following install methods.
+   1. Install the Temporal CLI with Homebrew.
+
+      ```bash
+      brew install temporal
+      ```
+
+   2. Install the Temporal CLI with cURL.
+
+      ```bash
+      curl -sSf https://temporal.download/cli.sh | sh
+      ```
+
+   3. Install the Temporal CLI for Windows.
+      1. Download and run the installer for the [latest release](https://github.com/temporalio/cli/releases/latest/).
+      2. Add the `temporal.exe` binary to your PATH.
+   4. Install the Temporal CLI from CDN.
```

Comment:
Hmm nice.
So if we're doing this, you might as well use these links in the quick install guide instead of directing people to GH releases.